src/main.js

```diff
@@ -94,6 +97,49 @@ function hideCoverButton() {
 function hideSaveCoverButton() {
   saveCoverButton.classList.add('hidden');
 }
+
+function storeCoverInputValue() {
+  var coverValue = coverInputValue.value;
+  covers.push(coverValue);
+}
+
+function storeTitleInputValue() {
+  var titleValue = titleInputValue.value;
+  titles.push(titleValue);
+}
+
+function storeDescriptorOneInputValue() {
+  var descriptorOneValue = firstDescriptorInputValue.value;
+  descriptors.push(descriptorOneValue);
+}
+
+function storeDescriptorTwoInputValue() {
+  var descriptorTwoValue = secondDescriptorInputValue.value;
+  descriptors.push(descriptorTwoValue);
+}
+
+function getUserData() {
```

Comment:
Nice work! This looks really clean. I like the way you did the nested functions!